code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -381,3 +381,19 @@
 	new /obj/item/gun/ballistic/automatic/pistol/m1911/kitchengun(src)
 	new /obj/item/ammo_box/magazine/m45/kitchengun(src)
 	new /obj/item/ammo_box/magazine/m45/kitchengun(src)
+
+
+/obj/item/storage/box/strange_seeds_10pack
+
+/obj/item/storage/box/strange_seeds_10pack/PopulateContents()
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)
+	new /obj/item/seeds/random(src)

```

Comment:
```
	for(var/i in 1 to 10)
		new /obj/item/seeds/random(src)
```
Maybe throw in a couple free seeds for convenience, It costs the whole traitor budget anyway.

Reply:
I was thinking of doing that in a loop, but I haven't really found any other container being populated with one and didn't want to stray away from that.
The main purpose of the 10pack is even more RNG magic of rolling a discount for it.

Reply:
Containers populated with loops already exist, you just haven't looked around enough, paper cups and lighting bulbs/tubes boxes use loops.
Regardless, there is no reason for not doing that. It saves lines.

Reply:
Fixed.